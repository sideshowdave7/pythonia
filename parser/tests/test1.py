def a():
 if (b < c):
  d
 else:
  e
def f():
 g; h
 i

