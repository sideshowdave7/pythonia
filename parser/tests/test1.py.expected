(program
 (def
  (a)
  (suite
   (cond ((comparison b ("<" c)) (suite (expr d))) (else (suite (expr e))))))
 (def (f) (suite (begin (expr g) (expr h)) (expr i))))

